CONTRIBUTING.md

```diff
@@ -103,14 +103,14 @@ time="2024-04-26T02:01:07Z" level=info msg="Generating README Documentation for
 
 ### Release
 
-1. Create a PR to update the version numbers and CHANGELOG.md
+1. Create a PR to update the version numbers, CHANGELOG.md, and Helm Docs (see above)
```

Comment:
Note: we're very good at doing the Helm Docs regen part (which I also forgot with v0.66.2)

Reply:
Note: this regenerates the README in https://github.com/pganalyze/collector/tree/main/contrib/helm/pganalyze-collector, especially the following part (which... you can see we've been forgetting to do so since 0.63.0 🙃 )
![image](https://github.com/user-attachments/assets/2d6fd48d-3567-4fc2-b6a1-d4c01e7fecab)